concourse/steps/release.mako

```diff
@@ -469,31 +468,31 @@ except:
   logger.warning('release will continue')
   traceback.print_exc()
 % else:
-release_notes_md = None
-full_release_notes_md = None
+release_notes_docs = []
 % endif
 
 tgt_ref = cnudie.util.target_oci_ref(component=component)
 
 if release_notes_md:
-  release_notes_octets = release_notes_md.encode('utf-8')
-  release_notes_digest = f'sha256:{hashlib.sha256(release_notes_octets).hexdigest()}'
+  tar_stream = b''.join(release_notes.tarutil.release_notes_docs_into_tarstream(release_notes_docs))'
+  tar_digest = f'sha256:{hashlib.sha256(tar_stream).hexdigest()}
+
   oci_client.put_blob(
     image_reference=tgt_ref,
-    digest=release_notes_digest,
-    octets_count=len(release_notes_octets),
-    data=release_notes_octets,
+    digest=tar_digest,
+    octets_count=len(tar_stream),
+    data=tar_stream,
   )
 
   component.resources.append(
     ocm.Resource(
-      name=release_notes.ocm.release_notes_resource_name,
+      name='release-notes-archive',
```

Comment:
I think we should keep publishing "old" release-notes in parallel for the time being - at least for a short transitioning phase - wdyt?